src/index.ts

```
@@ -1,12 +1,28 @@
 import createApp from "./lib/create-app";
 import index from "@/routes/index.route";
 const app = createApp();
 configureOpenAPI(app);
 const routes = [index];
 
+app.on(["POST", "GET"], "/api/auth/*", (c) => {
+  if (c.req.path === "/api/auth/use-session") {
+    const session = c.get("session");
+    const user = c.get("user");
+
+    if (!user) return c.body(null, 401);
```

ESLint

[error] Expect newline after if

(antfu/if-newline)